integration-tests/integration_test.go

```diff
@@ -576,12 +577,27 @@ func TestRingBuffer(t *testing.T) {
 }
 
 func TestUdpNetworkFlow(t *testing.T) {
-	if strings.Contains(config.VMInfo().Config, "rhel-8-4-sap") {
-		t.Skip("Skipping test on RHEL 8.4 SAP due to a verifier issue")
-	}
+	log.Info("VMInfo: %s", config.VMInfo)
 	if strings.Contains(config.VMInfo().Config, "fedora-coreos-stable") {
 		t.Skip("Skipping due to ROX-27673")
 	}
+	if strings.Contains(config.VMInfo().Config, "rhcos") {
+		t.Skip("Skipping due to ROX-32092")
+	}
+	if strings.Contains(config.VMInfo().Config, "rhel_rhel-8") {
+		t.Skip("Skipping due to ROX-32092")
+	}
+	if strings.Contains(config.VMInfo().Config, "rhel-sap") {
+		t.Skip("Skipping due to ROX-32092")
+	}
+	if strings.Contains(config.VMInfo().Config, "rhel-s390x") {
+		t.Skip("Skipping due to ROX-32092")
+	}
+	// rhel-ppc64le
+	if strings.Contains(config.VMInfo().Config, "rhel-88") {
+		t.Skip("Skipping due to ROX-32092")
+	}
+
```

Comment:
Something like this might be more readable, WDYT?
```suggestion
	skipped_vms := []string{
		"fedora-coreos-stable",
		"rhcos",
		"rhel_rhel-8",
		"rhel-sap",
		"rhel-s390x",
		"rhel-88", // ppc64le
	}
	for _, vm := range skipped_vms {
		if strings.Contains(config.VMInfo().Config, vm) {
			t.Skip("Skipping due to ROX-27673")
		}
	}

```

Also, since we are now skipping a large number of VMs here, we might want to circle back to this soon.

Reply:
Made a small change to keep tickets separate.